src/fides/api/ctl/routes/admin.py

```diff
@@ -16,7 +20,13 @@ class DBActions(str, Enum):
     reset = "reset"
 
 
-@router.post("/admin/db/{action}", tags=["Admin"])
+@router.post(
+    "/admin/db/{action}",
+    tags=["Admin"],
+    dependencies=[
+        Security(verify_oauth_client_cli, scopes=[scope_registry.DATABASE_RESET])
+    ],
+)
 async def db_action(action: DBActions) -> Dict:
```

Comment:
It's conceivable that a user would want to disable the ability to reset a DB on a running server entirely so I've added a ticket to implement that [here](https://github.com/ethyca/fides/issues/2319).